pkg/cmd/cli/cmd/whoami.go

```diff
@@ -57,6 +58,9 @@ func (o WhoAmIOptions) WhoAmI() (*userapi.User, error) {
 }
 
 func RunWhoAmI(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []string, o *WhoAmIOptions) error {
+	if len(args) > 0 {
+		return fmt.Errorf("no arguments should be provided")
```

Comment:
@fabianofranz this should be consistent in all commands that don't accept any arguments